### PR TITLE
5 get route assessments db

### LINKED ISSUE #5: Create a GET route to get the assessments from the DB
We will need to be able to feed our Assessment Content to the client-side of our application.

Create a GET route which feeds this content via our API, something like /assessments could work.

Tasks:
- [x] Refer to the [wiki](https://github.com/kahikatea-2019/EDAMAME/wiki/Server-Routes) to see how the routes may need to be setup.
- [x] Create the route.
- [x] Create db function.
- [x] Write tests for this route.
- [x] Write tests for db function.

### PR DESCRIPTION
**Pull request by mxbialostocki**
I need some input on this task - to be completed upon seed data release.

Closes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,12 +37,14 @@
     ]
   },
   "dependencies": {
+    "cheerio": "^1.0.0-rc.3",
     "express": "^4.16.4",
     "knex": "^0.16.5",
     "path": "^0.12.7",
     "react-router-dom": "^5.0.0",
     "sqlite3": "^4.0.6",
```
Comment (bryceknz) [inline package.json:46]: These testing frameworks (cheerio & supertest) should be dev dependencies.

Comment (mxbialostocki) [reply package.json:46]: Uninstall and re-install completed.

Comment (bryceknz) [reply package.json:46]: I believe they're both still regular dependencies? Remember to use `--save-dev` or `-D` when you run the install command.

**Comment by mxbialostocki** [comment]
Pulled in all dev changes to this branch including server.js updates

File: server/server.js [hunk 2]
```diff
@@ -6,12 +6,14 @@ const cohortsRoute = require('./routes/cohorts')
 const server = express()
 
 const starterRoute = require('./routes/starter')
+const assessmentsRoute = require('./routes/assessments')
 const authRoutes = require('./routes/auth')
 
 server.use(express.json())
 server.use(express.static(path.join(__dirname, './public')))
 
 server.use('/', starterRoute)
```
Comment (bryceknz) [inline server/server.js]: Does this routes file exist? It's breaking the code when I try to run it.

File: server/routes/assessments.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const router = express.Router()
+
+const db = require('../db/assessments')
+
+// GET route to run getAssessments db function else send error message
+router.get('/', (req, res) => {
+  res.send('Show me some good assessments')
```
Comment (bryceknz) [inline server/routes/assessments.js]: I think you will find that this line causes the route to fail. Once you call `res.send`, the exchange is complete! The second `res.send` on line 11 won't do anything :(

**Review by chrislaupama — DISMISSED** [review]
Need to remove the first response for when you hit the API because it just sends text not the DB json response we want.

You also need to work on getting that json data working as it's currently displaying an error.

**Comment by mxbialostocki** [comment]
The function now returns data at /assessments

**Comment by mxbialostocki** [comment]
Server.js merged, tests all passing

**Comment by mxbialostocki** [comment]
Route and DB tests are both passing now

**Review by bryceknz — APPROVED** [review]

**Pull request merged**